enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/errorhandling/OnboardingProcessLimitException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2021 Wultra s.r.o.
```

Comment:
2022

Reply:
Ah, right, copy-paste error. My template should be fine already.